stardoc/BUILD

```diff
@@ -54,3 +54,25 @@ java_import(
     jars = ["stardoc_binary.jar"],
     visibility = ["//visibility:private"],
 )
+
+java_binary(
+    name = "renderer",
+    jvm_flags = [
+        # quiet warnings from com.google.protobuf.UnsafeUtil,
+        # see: https://github.com/google/protobuf/issues/3781
+        # TODO(cparsons): Remove once Stardoc has the fix.
+        "-XX:+IgnoreUnrecognizedVMOptions",
+        "--add-opens=java.base/java.nio=ALL-UNNAMED",
+        "--add-opens=java.base/java.lang=ALL-UNNAMED",
+    ],
+    main_class = "com.google.devtools.build.skydoc.renderer.RenderMain",
```

Comment:
I have another fun one that took me some time to debug ;)

This is the wrong class name...